x/wasm/types/codec.go

```diff
@@ -70,6 +70,7 @@ var (
 )
 
 func init() {
+	RegisterLegacyAminoCodec(legacy.Cdc)
```

Comment:
this seems necessary for authz + ledger

https://github.com/cosmos/cosmos-sdk/blob/v0.46.0-rc1/x/bank/types/codec.go#L47